### ISSUE TITLE
will open source the implementation of cnn with opencl

### ISSUE THREAD

**Issue by xingjinglu** [JC1DA/DeepSense#2]

**Comment by JC1DA** [comment]
HI, It's already in Assets directory :)

Check this path: app/src/main/assets

**Comment by sivagnanamn** [comment]
Does this implementation use "Batchnorm" layer? I couldn't find any kernel that does the batch norm operation.
